ext/zlib/tests/zlib_get_coding_type.phpt

```diff
@@ -0,0 +1,28 @@
+--TEST--
+string zlib_get_coding_type ( void );
+--CREDITS--
+marcosptf - <marcosptf@yahoo.com.br> - @phpsp - sao paulo - br
+--SKIPIF--
+<?php
```

Comment:
zlib_get_coding_type() is available as of PHP 4.3.2, so the --SKIPIF-- section should be changed to reflect whether or not the zlib extension is loaded e.g.:

if (!extension_loaded('zlib')) {
 print 'skip: zlib extension not available';
}

or similar